unittests/32Bit_ASM/CMakeLists.txt

```diff
@@ -83,6 +83,9 @@ foreach(ASM_SRC ${ASM_SOURCES})
 
     set(TEST_NAME "${TEST_DESC}/Test_32Bit_${REL_TEST_ASM}")
     string(REPLACE " " ";" ARGS_LIST ${ARGS})
+
+    list(APPEND ARGS_LIST "--")
```

Comment:
The logic `ARGS_LIST` is involved in is complex enough already. Can we change the below to this instead?

```
${LAUNCH_PROGRAM}
${ARGS_LIST}
"--"
"{OUTPUT_NAME}..."
```
This makes the structure of the arguments much clearer than hiding the separator in a variable.